app/mutations/application/createApplicationRevisionStatusMutation.ts

```diff
@@ -9,21 +9,10 @@ import BaseMutation from 'mutations/BaseMutation';
 const mutation = graphql`
   mutation createApplicationRevisionStatusMutation(
     $input: CreateApplicationRevisionStatusInput!
-    $version: String
   ) {
     createApplicationRevisionStatus(input: $input) {
       applicationRevisionStatus {
         id
-        versionNumber
-        applicationRevisionByApplicationIdAndVersionNumber {
-          applicationByApplicationId {
-            id
-            applicationRevisionStatus(versionNumberInput: $version) {
-              id
-              applicationRevisionStatus
-            }
-          }
-        }
```

Comment:
Simplified a bit here, as this mutation is now only used in the `SubmitApplication` button, and none of the response is used to render anything (it just [navigates the user to the submission confirmation page](https://github.com/bcgov/cas-ciip-portal/blob/1906b20a48c03bc16da2f10fd90f73d446ab9300/app/components/SubmitApplication.tsx#L37) afterward).